util/dockernode.go

```diff
@@ -196,6 +203,62 @@ func (dn *DockerNode) setLatency(val string) error {
 	return cnet.SetLink(ifn, settings)
 }
 
+func (dn *DockerNode) setJitter(val string) error {
+	dur, err := time.ParseDuration(val)
+	if err != nil {
+		return err
+	}
+
+	ifn, err := dn.getInterfaceName()
+	if err != nil {
+		return err
+	}
+
+	settings := &cnet.LinkSettings{
+		Jitter: int(dur.Nanoseconds() / 1000000),
+	}
+
+	return cnet.SetLink(ifn, settings)
+}
+
+// set bandwidth (expects Mbps)
+func (dn *DockerNode) setBandwidth(val string) error {
+	bw, err := strconv.ParseFloat(val, 32)
+	if err != nil {
+		return err
+	}
+
+	ifn, err := dn.getInterfaceName()
+	if err != nil {
+		return err
+	}
+
+	settings := &cnet.LinkSettings{
+		Bandwidth: int(bw / 1000000),
```

Comment:
Is this right? The library takes bits per second, if i'm understanding it properly. If our input is in mbps, and we divide by 1000000 then we will be giving it a value of terabits per second.

Reply:
Ha, right, did that backwards. Good catch